### ISSUE TITLE
Jetbrains product pressed "backspace" is invalid in chat input

### ISSUE THREAD

**Issue by MikeLINGxZ** [continuedev/continue#3555]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: macos 15.0 (24A335)
- Continue version: 0.0.83
- IDE version: Jetbrains all 2024.3
- Model: all
```

### Description

Jetbrains product pressed "backspace" is invalid in chat input

**Comment by sestinj** [comment]
Hey @MikeLINGxZ, I think we've solved this in version 0.0.84. It is currently available in EAP: https://docs.continue.dev/troubleshooting#jetbrains-1

**Comment by acehoss** [comment]
Just installed Continue in Rider 2024.3 on macOS 15.1.1 and also experienced this issue. I can type in the chat just fine, but I can't backspace. I added the EAP repo and updated to 0.0.85 but the behavior is unchanged.

TL;dr: Appears to be still broken in 0.0.85.

**Comment by N3rd00d** [comment]
It still occurs in macOS Rider 2024.3 0.0.87.

**Comment by deancanen** [comment]
I have a workaround for the Rider IntelliJ keymap: Remove backspace from "Remove Selected Tests (Tool Window)" and backspace will start working in Continue again.

**Comment by miguelzakharia** [comment]
> I have a workaround for the Rider IntelliJ keymap: Remove backspace from "Remove Selected Tests (Tool Window)" and backspace will start working in Continue again.

Confirmed. Thanks for the workaround!

**Comment by jamesneal21** [comment]
Still broken in 1.0.21!
